android/app/build.gradle

```diff
@@ -24,6 +25,10 @@ buildscript {
   // Add a parameter to force Firebase.
   ext.googleFirebaseServicesEnabled = project.hasProperty('firebase') ?: googleFirebaseServicesDefault
 
+  // Enable Android Auto for all debug and beta flavors except fdroid .
+  def androidAutoEnabledDefault = (isDebug || isBeta) && !isFdroid
```

Comment:
Should we have some flag `-PenableAA` or `-PdisableAA` to force build with (or without) AA?

Reply:
Yes, it would be nice to have. Unfortunally, this patch doesn't work properly because AndroidManifest.xml from Auto flavor is not merged into the main manifest. I am searching for alternative solutions.

Reply:
-Pandroidauto=true | -Pandroidauto=false